### ISSUE TITLE
detach() or stop_gradient() equivalent?

### ISSUE THREAD

**Issue by anirbanl** [google/jax#2025]
If we want certain jax array to be invariant to gradient operator, what is the jax equivalent that we can use (in PyTorch detach() function or in tensorflow stop_gradient() operator works)?

**Comment by shoyer** [comment]
jax.lax.stop_gradient does this: https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.stop_gradient.html#jax.lax.stop_gradient

**Comment by anirbanl** [comment]
Thanks!
